lib/routes.js

```diff
@@ -0,0 +1,27 @@
+var log    = require('../middleware/logger').child({
+  level : 'info',
+  component : '[ROUTES]'
+});
+
+function registerRoutes(router){
+  router.get('/', function (req, res) {
```

Comment:
everything looks good ... just as a pointer you should be consistent with either using req, res / request, response throughout app